src/common/summarize-profile.js

```diff
@@ -191,10 +203,10 @@ function summarizeSampleCategories(summary, fullCategoryName) {
  * @param {object} summary - The object that summarizes the times of the samples.
  * @return {array} The summary with percentages.
  */
-function calculateSummaryPercentages(summary) {
-  const rows = Object.entries(summary);
+function calculateSummaryPercentages(summary: Summary) {
+  const rows:Array<[string, number]> = objectEntries(summary);
```

Comment:
I'm surprised you needed this type `Array<[string, number]>`. should be inferred if `objectEdntries` has a return